assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -85,7 +104,7 @@ export default function UserDimensionsPieChart( { dimensionName, sourceLink, loa
 				}
 			} );
 		}
-	}, [ chartID, dimensionName, setValues ] );
+	}, [ chartID, dimensionName, dimensionValue, report, setValues ] );
```

Comment:
We probably shouldn't use an object as a dependency item for a hook because it can break memoization. Let's better convert it to a string using the `JSON.stringify` function.

Reply:
@eugene-manuilov I thought about that, but it shouldn't be an issue here? The report object returned by the `getReport` selector is always the same for the same set of request args.